src/document.js

```diff
@@ -20,8 +20,20 @@ const PostMessages = new PostMessageService({
 
 const showLoadingIndicator = () => document.getElementById('loadingContainer').classList.add('icon-loading')
 const hideLoadingIndicator = () => document.getElementById('loadingContainer').classList.remove('icon-loading')
+const showLoading = () => {
+	if (OC.appswebroots.richdocumentscode && Config.get('urlsrc').indexOf('proxy.php') >= 0) {
+		$.get(OC.appswebroots.richdocumentscode + '/proxy.php?status').done(function(val) {
+			if (val && val.status && val.status === 'starting') {
+				$('#proxyLoadingIcon').addClass('icon-loading-small')
```

Comment:
We're actually trying to get rid of jquery :wink: please switch that to some plain javascript 

https://developer.mozilla.org/de/docs/Web/API/Document/getElementById
http://youmightnotneedjquery.com/#add_class
http://youmightnotneedjquery.com/#set_text

Reply:
OK!